Resources/Prototypes/SS220/Entities/Clothing/Sponsor/clothing.yml

```
@@ -0,0 +1,281 @@
+
+- type: entity
+  parent: ClothingSponsorOuterStorageBase
+  id: ClothingSponsorDante
+  name: вишнёвый плащ
+  description: Але?
+  components:
+  - type: Sprite
+    sprite: SS220/Clothing/Sponsor/dante.rsi
+  - type: Clothing
+    sprite: SS220/Clothing/Sponsor/dante.rsi
+
+- type: entity
+  parent: ClothingSponsorMaskBase
+  id: ClothingSponsorDallas
+  name: демократическая маска
+  description: Вошли и вышли, дело пяти минут.
+  components:
+  - type: Sprite
+    sprite: SS220/Clothing/Sponsor/dallas/mask.rsi
+  - type: Clothing
+    sprite: SS220/Clothing/Sponsor/dallas/mask.rsi
+  - type: BreathMask
+  - type: IdentityBlocker
+
+- type: entity
+  parent: ClothingSponsorHeadBase
+  id: ClothingSponsorMafiaHat
+  name: розовая шляпа
+  description: Я - Зубен...
+  components:
+  - type: Sprite
+    sprite: SS220/Clothing/Sponsor/mafia/hat.rsi
+  - type: Clothing
+    sprite: SS220/Clothing/Sponsor/mafia/hat.rsi
+
+- type: entity
+  parent: ClothingSponsorUniformBase
+  id: ClothingSponsorMafiaSuit
+  name: костюм прекрасного человека
+  description: Розово и стильно.
+  components:
+  - type: Sprite
+    sprite: SS220/Clothing/Sponsor/mafia/suit.rsi
+  - type: Clothing
+    sprite: SS220/Clothing/Sponsor/mafia/suit.rsi
+
+- type: entity
+  parent: ClothingSponsorShoesBase
+  id: ClothingSponsorMafiaBoots
+  name: ботинки прекрасного человека
+  description: Цок-цок.
+  components:
+  - type: Sprite
+    sprite: SS220/Clothing/Sponsor/mafia/boots.rsi
+  - type: Clothing
+    sprite: SS220/Clothing/Sponsor/mafia/boots.rsi
+
+- type: entity
+  parent: ClothingSponsorHeadBase
+  id: ClothingSponsorMafiaBlackHat
+  name: чёрная шляпа
+  description: Меня зовут...
+  components:
+  - type: Sprite
+    sprite: SS220/Clothing/Sponsor/mafia/black/hat.rsi
+  - type: Clothing
+    sprite: SS220/Clothing/Sponsor/mafia/black/hat.rsi
+
+- type: entity
+  parent: ClothingSponsorUniformBase
+  id: ClothingSponsorMafiaBlackSuit
+  name: тёмный костюм прекрасного человека
+  description: Темно и стильно.
+  components:
+  - type: Sprite
+    sprite: SS220/Clothing/Sponsor/mafia/black/suit.rsi
+  - type: Clothing
+    sprite: SS220/Clothing/Sponsor/mafia/black/suit.rsi
+
+- type: entity
+  parent: ClothingSponsorShoesBase
+  id: ClothingSponsorMafiaBlackBoots
+  name: тёмные ботинки прекрасного человека
+  description: Цок-цок.
+  components:
+  - type: Sprite
+    sprite: SS220/Clothing/Sponsor/mafia/black/boots.rsi
+  - type: Clothing
+    sprite: SS220/Clothing/Sponsor/mafia/black/boots.rsi
+
+- type: entity
+  parent: ClothingSponsorUniformBase
+  id: ClothingSponsorMaidSuit
+  name: чёрный костюм горничной
+  description: Почти не пачкается.
+  components:
+  - type: Sprite
+    sprite: SS220/Clothing/Sponsor/maid/suit.rsi
+  - type: Clothing
+    sprite: SS220/Clothing/Sponsor/maid/suit.rsi
+
+- type: entity
+  parent: ClothingSponsorHeadBase
+  id: ClothingSponsorMaidHat
+  name: гребень горничной
+  description: Самое мужское одеяние
```

```suggestion
  description: Самое мужественное одеяние.
```
может ты это имела ввиду? мужское - предназначенное для мужчин, мужественное - отражающее мужественность